app/schemas/MENU.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Why do we need a `Menu` collection? Conceptually I can't quite see the relevance. If, for example, you want all the food that say Mcdonald's sells, you can query `select * from food where food.restaurantUid === theUidOfMcDonalds`.

Placing the menu in the middle makes this a triple "join" (quotes because joins don't exist in noSQL), which means 3 separate client-side fetches.

If you want to have say separate breakfast, lunch and dinner menus, we can add that property on the `food`

Reply:
Im not so sure about having this Menu one, however, I would say using a nosql db, having a menu collection might makes things easier for us